_includes/manuals/2.1/3.3.1_rpm_packages.md

```diff
@@ -6,7 +6,7 @@ Foreman is packaged for the following RPM based distributions:
 
 **Note:** The RPM packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
 
-For most users, it's highly recommended to use the [installer](manuals/{{page.version}}/index.html#3.2ForemanInstaller) as the packages only provide the software and a standalone Foreman service.  The installer installs these packages, then additionally configures Foreman to run under Apache and Passenger with PostgreSQL, plus can configure a complete Puppet setup integrated with Foreman.
+For most users, it's highly recommended to use the [installer](manuals/{{page.version}}/index.html#3.2ForemanInstaller) as the packages only provide the software and a standalone Foreman service.  The installer installs these packages, then additionally configures Foreman to run under Apache and Puma with PostgreSQL, plus can configure a complete Puppet setup integrated with Foreman.
```

Comment:
To me 'run under Apache' feels unnatural since Puma is the application server that actually runs it. Does something like this make sense?
```suggestion
For most users, it's highly recommended to use the [installer](manuals/{{page.version}}/index.html#3.2ForemanInstaller) as the packages only provide the software and a standalone Foreman service.  The installer installs these packages, then additionally configures Foreman to run under Puma with Apache as a reverse proxy and PostgreSQL, plus can configure a complete Puppet setup integrated with Foreman.
```